packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -69,7 +87,26 @@ export interface NodeRangePath {
 	count: number;
 }
 
-export const makeEditGenerator = (): AsyncGenerator<Operation, FuzzTestState> => {
+export interface EditGeneratorOpWeights {
+	insert: number;
+	delete: number;
+	setPayload: number;
+	start: number;
+	commit: number;
+	abort: number;
```

Comment:
I like this idea of configurable weights.
I think making each number optional would be extra nice for two reasons:
- This way users can just omit whatever option they don't want to generate operations for.
- It won't break the code each time we add a new option.

Reply:
made each weight optional